### DISCUSSION TITLE
How to use slots in custom components?

### DISCUSSION THREAD

**Discussion by san-kumar** [formkit/formkit#1078] [General]
First of all this is a great resource. 

To encourage re-usability I'm trying to create a custom component as shown in the docs [here](https://formkit.com/guides/create-a-custom-input). 

But unfortunately I'm not able to make the slots works. It seems that any children I pass between the `<FormKit><child/></FormKit>` does not replace inside the Custom component's `<slot></slot>` tag.

Does anyone have any idea how to make it work or is this a bug or unsupported?

Here is a playground link to show you what I mean

https://formkit.link/be2c9914ded5bbf39844e8c04c78cab6

Thanks for the help!

**Comment by Finrod927** [comment]
I had the same question and got an answer when creating a issue: https://github.com/formkit/formkit/issues/1365#issuecomment-2122807358
